### PR TITLE
Swap out miniforge for mambaforge

### PR DESCRIPTION
**Pull request by HarshCasper**
Fixes https://github.com/nebari-dev/nebari-docker-images/issues/3

### CODE REVIEW

File: scripts/install-conda.sh [hunk 1]
```diff
@@ -1,25 +1,9 @@
...
 sh ./miniconda.sh -b -p /opt/conda
-rm miniconda.sh miniconda.checksum
-
-# install mamba and clean up
-conda install -y -c conda-forge mamba
-mamba clean -afy
-find /opt/conda/ -follow -type f -name '*.a' -delete
-find /opt/conda/ -follow -type f -name '*.js.map' -delete
-
-ln -s /opt/conda/etc/profile.d/conda.sh /etc/profile.d/conda.sh
```
Comment (iameskild) [inline scripts/install-conda.sh:22]: Why was this line removed?

File: scripts/install-conda.sh [hunk 2]
```diff
@@ -1,25 +1,9 @@
...
-wget --quiet -O miniconda.sh https://repo.anaconda.com/miniconda/Miniconda3-$CONDA_VERSION-Linux-x86_64.sh
-
-echo "${CONDA_SHA256} miniconda.sh" > miniconda.checksum
-
-if [ $(sha256sum -c miniconda.checksum | awk '{print $2}') != "OK" ]; then
-   echo Error when testing checksum
-   exit 1;
-fi
 
+curl -sL -o miniconda.sh "https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-$(uname)-$(uname -m).sh"
```
Comment (iameskild) [inline scripts/install-conda.sh:4]: In previous builds, we are pinning the version using `CONDA_VERSION` (inside of the Dockerfile.jupyterlab), perhaps its worthwhile pinning the version here as well. In the original issue, Tania also recommended that we upgrade to python 3.8.

**Review by iameskild — CHANGES_REQUESTED** [review]
Hey @HarshCasper, thanks for handling this! I made a few comments that we should investigate before merging.

File: scripts/install-conda.sh (hunk 2 above)
Comment (HarshCasper) [reply scripts/install-conda.sh]: The recommendation to upgrade to Python 3.8 was to be done in a different PR if I remember correctly.

**Review by iameskild — APPROVED** [review]
Great work @HarshCasper!! Thanks for addressing the comments above!
